### ISSUE TITLE
Measure response time

### ISSUE THREAD

**Issue by as51340** [jepsen-io/jepsen#594]
Hi,

Is there a way to measure the time needed for database to process a request directly using Jepsen? I am trying to see if response time degrades in the fault-injected environment...

Andi

**Comment by aphyr** [comment]
Yes--response times are graphed in `latency-raw.png`, and you'll also see overlays when the nemesis was active.
